calico/src/main/scala/calico/html/Children.scala

```diff
@@ -34,21 +34,21 @@ import scala.scalajs.js
 final class Children[F[_]] private[calico]:
   import Children.*
 
-  inline def <--(
+  @inline def <--(
```

Comment:
`inline` applies at compile-time, but it has some restrictions. For example you cannot inline a private constructor.

`@inline`applies at link-time, and has no privacy-related restrictions.

CI says `+3B 0% total files change` so seems like it didn't make a difference.